user_manual/docs/firmware.md

```diff
@@ -5,39 +5,47 @@ Updating Firmware
 
 # Overview {#update-overview}
 
-The **Firmware** is the code that runs on the board.
+The **Firmware** is the OLE code that runs on the device during normal operating
+conditions.
 
-The **Bootloader** is a mode that allows the firmware to be written to the board
+The **Bootloader** is a mode that allows the firmware to be written to the
```

Comment:
And the code that enables it. Perhaps try and be more consistent with the above.